src/julia/Integer.jl

```diff
@@ -218,6 +218,15 @@ function ppio(a::T, b::T) where T <: Integer
    return c, n
 end
 
+###############################################################################
+#
+#   Primality test
+#
+###############################################################################
+
+isprobable_prime(x::Integer, reps::Integer=25) =
```

Comment:
I don't personally like using the one line function syntax on two lines, by the way. If it doesn't fit on one line, it should have function and end. It's just too easy to get confused by it.

Reply:
OK, I will try to remember.